static/js/recent_topics_ui.js

```diff
@@ -252,38 +298,83 @@ export function process_messages(messages) {
     }
 }
 
-function format_topic(topic_data) {
-    const last_msg = message_store.get(topic_data.last_msg_id);
-    const stream = last_msg.stream;
-    const stream_id = last_msg.stream_id;
-    const stream_info = sub_store.get(stream_id);
-    if (stream_info === undefined) {
-        // stream was deleted
-        return {};
-    }
-    const topic = last_msg.topic;
+function format_message(msg_data) {
```

Comment:
Let's call this `format_conversation(conversation_data) {`; we're doing computations around a conversation, not a message, here.

Reply:
I have changed the function name to `format_conversation(conversation_data)` and `msg_info` to `context`.